top-app-bar/Flutter/Flutter-top-app-bar.md

```diff
@@ -54,21 +56,26 @@ The top app bar provides content and actions related to the current screen. It
 `AppBar`
 * [Class definition](https://api.flutter.dev/flutter/material/AppBar-class.html)
 * [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/app_bar.dart)
+* [Dartpad Demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/top-app-bar/regular&gh_ref=develop)
 
 The following example shows a top app bar with a page title, a navigation icon, two action icons, and an overflow menu:
 
-![insert screenshot here]()
-
-```
-The source code for the regular top app bar should include the following:
-* A purple container
-* A navigation icon
-* A title "Page title" in white
-* Two action items:
-    * favorite icon
-    * search icon
-* Overflow menu
-* The top app bar should remain in place while the user scrolls
+![assets/top_app_bar_screenshot.png](assets/top_app_bar_screenshot.png)
```

Comment:
Replace the alt text with "Top app bar with nav icon, page title, favorite icon, and search icon"

Reply:
Done